src/components/text.js

```diff
@@ -161,7 +163,10 @@ module.exports.Component = registerComponent('text', {
     }
 
     // Shader has changed. Create a shader material.
-    NewShader = createShader(this.el, data.shader, shaderData);
+    // For convenience, if font matches MSDF naming convention, change sdf to msdf.
+    shaderName = data.shader;
+    if (data.shader === 'sdf' && data.font.indexOf('-msdf.') >= 0) { shaderName = 'msdf'; }
```

Comment:
A string check is fragile. The shader is configurable, maybe just let people set the shader. But we can switch it if it matches a stock MSDF font.